test/e2e/cluster/cluster.go

```diff
@@ -1,5 +1,28 @@
 package cluster
 
+// import (
+// 	"errors"
+// 	"fmt"
+// 	"reflect"
+// 	"time"
+
+// 	. "github.com/onsi/ginkgo"
+// 	. "github.com/onsi/gomega"
+// 	kruise "github.com/openkruise/kruise-api/apps/v1beta1"
+// 	corev1 "k8s.io/api/core/v1"
+// 	k8serr "k8s.io/apimachinery/pkg/api/errors"
+// 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+// 	k8syaml "k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+// 	"k8s.io/apimachinery/pkg/types"
+// 	"sigs.k8s.io/controller-runtime/pkg/client"
+// 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+// 	kvrocksv1alpha1 "github.com/RocksLabs/kvrocks-operator/api/v1alpha1"
+// 	"github.com/RocksLabs/kvrocks-operator/pkg/client/kvrocks"
+// 	"github.com/RocksLabs/kvrocks-operator/pkg/resources"
+// 	"github.com/RocksLabs/kvrocks-operator/test/e2e/suite"
```

Comment:
@jiayouxujin We should remove the code directly since we're using the version control service and can take it back if we want.

Reply:
Ok, I will remove the unnecessary code.